moveit_core/kinematics_base/include/moveit/kinematics_base/kinematics_base.h

```diff
@@ -612,33 +614,74 @@ class KinematicsBase
   template <typename T>
   inline bool lookupParam(const std::string& param, T& val, const T& default_val) const
   {
-    ros::NodeHandle pnh("~");
-    if (pnh.hasParam(group_name_ + "/" + param))
-    {
-      val = pnh.param(group_name_ + "/" + param, default_val);
-      return true;
+    // ros::NodeHandle pnh("~");
+
+    auto node = rclcpp::Node::make_shared("lookupparam");
```

Comment:
use underscores for node names to increase readability

Reply:
Cool, thanks a415ab5ef046bb2081f4a967a083ae651c2e618c